niceman/support/starcluster/sshutils.py

```diff
@@ -517,10 +517,10 @@ def _get_output(self, channel, silent=True, only_printable=False):
                     line = ''.join(c for c in line if c in string.printable)
                 if line != '':
                     output.append(line)
-                    print line,
+                    print(line),
```

Comment:
oh -- starcluster is not python3 compatible?  we are asking for trouble I am afraid... we might want to just futurize its source code or smth like that at least...